pkgs/development/haskell-modules/configuration-hackage2nix/main.yaml

```diff
@@ -125,6 +125,7 @@ extra-packages:
   - ShellCheck == 0.9.0                 # 2024-03-21: pinned by haskell-ci
   - versions < 6                        # 2024-04-22: required by spago-0.21
   - fsnotify < 0.4                      # 2024-04-22: required by spago-0.21
+  - th-desugar == 1.16                  # 2024-06-28: preserve for GHC 9.8
```

Comment:
I hope this is the correct way to force a specific version into hackage-packages.nix :+1: